### ISSUE TITLE
Cmake latest stable is 3.21, Will it causes problems when we use 3.22rc version?

### ISSUE THREAD

**Issue by Femtometer** [IARSystems/cmake-tutorial#7]

**Comment by felipe-iar** [comment]
This is a good question.

CMake 3.22.1 was initially supposed to be released on November 10th.
The CMake authors postponed the release to [November 24th](https://gitlab.kitware.com/cmake/cmake/-/milestones/91).

CMake 3.22.1 will contain a number of fixes specific to the IAR C/C++ Toolchains.
According to the tutorial's [__requirements__](https://github.com/iarsystems/cmake-tutorial#required-tools), It is not guaranteed it  can be run on previous versions of CMake.

Please feel free to try this tutorial using CMake __3.22.0-rc2__. 
It is possible to deploy in parallel with other versions (i.e. [__cmake-3.22.0-rc2-windows-x86_64.zip__](https://github.com/Kitware/CMake/releases/download/v3.22.0-rc2/cmake-3.22.0-rc2-windows-x86_64.zip)). 
Release candidates usually offers the most approximate experience you would have with the released version.

Generally speaking, the purpose of Release Candidate versions is that, If you face a bug specifically related to the latest version of CMake, we all have the chance to report directly at the [__official CMake development repository__](https://gitlab.kitware.com/cmake/cmake). In other words, now probably is the best time for you to try it.

From my end, I did not detect any blocking issues, although, I would __not__ recommend using _release candidate_ versions of any piece of software into actual production.

I hope it helps.

Please feel free to open a [new issue](https://github.com/IARSystems/cmake-tutorial/issues/new) if you have another question in the future about this [CMake tutorial for IAR C/C++ Compiler users](https://github.com/iarsystems/cmake-tutorial). 
Thank you.
